on_the_project.md

```diff
@@ -0,0 +1,242 @@
+6. Making a sketch of what you want to do over the next week or two for the
+   project.
+
+## Github practice, questions
+
+You are going to get this file, and several others, as a *pull request* (PR) to
+your repository.
+
+Your first job is to use this PR to ask questions of us, your instructors.
+
+What we propose you do, is use the PR interface to ask for clarification about
+the task, or the project.  You can enter comments in the PR interface, or go
+the "Files changed" tabs, and click on individual lines to add comments or
+questions about specific lines in the file.
+
+Use the tag `@nipraxis-fall-2022/instructors` to point us to your questions.
+
+Once you are happy you've understood the task, merge this PR.
+
+## On Markdown
+
+The file is in Markdown format, and you will be writing an analysis plan, also
+in Markdown.
+
+Markdown is a *markup language*.  A Markdown file is a conventional text file
+that you can open in any text editor.  The special aspect of a Markdown file is
+the *markup*.   Markup consists of special bits of text that specify
+*formatting* of the text.  For example, in order to make a word in **bold**
+text, using Markdown markup, you put two asterisks either side of the text you
+want to be in bold.  When you want a properly formatted version of your
+Markdown file, you convert it to the formatted version, using a *Markdown
+renderer*.  A Markdown renderer is some system that can interpret the Markdown
+markup and display the text as you intended, with bold text as bold, headings
+as headings and so on.
+
+There are very many Markdown renderers, but the Github site is one.   When you
+put a `.md` file into your repository, like this one, and then navigate to the
+relevant file in the Github web interface, you will see that Github has
+*rendered* the Markdown formatting, showing bold as **bold**, headings as
+headings, and so on.
+
+Markdown has become the standard way of writing text files with markup, and you
+will see it everywhere on systems that programmers use, such as Github, and in
+the Jupyter notebook.
+
+Markdown has many dialects, meaning that there is some markup that every
+Markdown renderer understands, such as **bold**, and other markup that only
+some renderers understand.  The Markdown that every renderer understands is
+called [standard Markdown](https://www.markdownguide.org/basic-syntax). Github
+has its own dialect of Markdown, called [Github flavored
+Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github).
+You can usually stick to the standard stuff, but you may need to consult the
+Github documents if you want to do something slightly more fancy, like a table.
+
+## Making sure you're ready
+
+To be ready to get going on your project you need to make sure you have merged these three PRs:
+
+* "add-dvars"
+* "Add machinery to install module directory."
+* "Fix use of Path in find_outliers script"
```

Comment:
@matthew-brett We didn't get this PR, but I guess it's about the bug we found in c240d0b1941ca7b782dbacb6c2cb7a193244fbe4?

Reply:
I guess so. Also for "add-dvars" it was our PR #7.